src/tools/kata-ctl/src/ops/check_ops.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2022 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use crate::arch;
+use crate::check;
+use crate::ops::version;
+
+use crate::args::{
+    CheckCommand,
+    CheckSubCommand,
+    IptablesCommand,
+    MetricsCommand
+};
+
+const NAME: &str = "kata-ctl";
+
+pub fn handle_check_command(checkcmd: CheckCommand) {
+    let command = checkcmd.command;
+
+    match command {
+        CheckSubCommand::All => {
+            // run architecture-specific tests
+            let _all_checks = arch::check();
+
+            // run code that uses network checks
+            let _network_checks = check::run_network_checks();
+        }
+
+        CheckSubCommand::NoNetworkChecks => {
+            // run architecture-specific tests
+            let _all_checks = arch::check();
+        }
+
+        CheckSubCommand::CheckVersionOnly => {
+            let _version_check = check::check_version();
+        }
+    }
+}
+
+pub fn handle_check_volume() {
+    unimplemented!("run volume command");
```

Comment:
For now, the unimplemented handlers can just return `Ok`:

```suggestion
    unimplemented!("run volume command");
    Ok(())
```

Reply:
thanks @jodh-intel , fixed in the next PR attempt